python/pyarrow/tests/test_pandas.py

```diff
@@ -1071,13 +1071,12 @@ def test_python_datetime_with_timezone_tzinfo(self):
         pytz = pytest.importorskip("pytz")
         from datetime import timezone
 
-        if Version(pd.__version__) > Version("0.25.0"):
-            # older pandas versions fail on datetime.timezone.utc (as in input)
-            # vs pytz.UTC (as in result)
-            values = [datetime(2018, 1, 1, 12, 23, 45, tzinfo=timezone.utc)]
-            # also test with index to ensure both paths roundtrip (ARROW-9962)
-            df = pd.DataFrame({'datetime': values}, index=values)
-            _check_pandas_roundtrip(df, preserve_index=True)
+        # older pandas versions fail on datetime.timezone.utc (as in input)
+        # vs pytz.UTC (as in result)
```

Comment:
nit: Seems like this comment can also go, or rephrased, or the test can go if it was specific to pre-1.0.0 pandas?